index.html

```diff
@@ -4,10 +4,10 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Pelican Town FarmShop</title>
-    <link rel="stylesheet" href="index.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com"/>
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin/>
     <link href="https://fonts.googleapis.com/css2?family=Jaro:opsz@6..72&family=VT323&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="index.css" />

```

Comment:
Make sure you add your own styles after any external fonts / styles are added (loaded).